src/modules/navigator/rtl_params.c

```diff
@@ -136,6 +136,16 @@ PARAM_DEFINE_INT32(RTL_TYPE, 0);
  */
 PARAM_DEFINE_INT32(RTL_CONE_ANG, 0);
 
+/**
+ * Maximum allowed RTL flight in minutes
+ *
+ * This is used to determine when the vehicle should be switched to RTL due to low battery.
+ * Note, particularly for multirotors this should reflect flight time at cruise speed, not while stationary
+ *
+ * @group Commander
```

Comment:
units

Reply:
Is minutes a problem? It is documented in the string. It's just that a number like 15 or 30 is much easier to understand for users than 900 or 1800 if I used seconds instead.

Reply:
Minutes is fine, I mean add the unit metadata (@unit).